lusid-sdk-python/tests/InstrumentLoader.py

```diff
@@ -0,0 +1,93 @@
+                                                                          test_cls.CUSTOM_INTERNAL_SCHEME: "INTERNAL_ID_1"},
+                                                             properties=[isin_vod, sedol_vod])
+
+        isin_barc = models.InstrumentProperty(test_cls.ISIN_PROPERTY_KEY, models.PropertyValue("GB0031348658"))
+        sedol_barc = models.InstrumentProperty(test_cls.SEDOL_PROPERTY_KEY, models.PropertyValue("3134865"))
+
+        instrument_definition2 = models.InstrumentDefinition(name="BARCLAYS PLC",
+                                                             identifiers={test_cls.FIGI_SCHEME: "BBG000C04D57",
+                                                                          test_cls.CUSTOM_INTERNAL_SCHEME: "INTERNAL_ID_2"},
+                                                             properties=[isin_barc, sedol_barc])
+
+        isin_grid = models.InstrumentProperty(test_cls.ISIN_PROPERTY_KEY, models.PropertyValue("GB00BDR05C01"))
+        sedol_grid = models.InstrumentProperty(test_cls.SEDOL_PROPERTY_KEY, models.PropertyValue("BDR05C0"))
+
+        instrument_definition3 = models.InstrumentDefinition(name="NATIONAL GRID PLC",
+                                                             identifiers={test_cls.FIGI_SCHEME: "BBG000FV67Q4",
+                                                                          test_cls.CUSTOM_INTERNAL_SCHEME: "INTERNAL_ID_3"},
+                                                             properties=[isin_grid, sedol_grid])
+
+        isin_sains = models.InstrumentProperty(test_cls.ISIN_PROPERTY_KEY, models.PropertyValue("GB00B019KW72"))
+        sedol_sains = models.InstrumentProperty(test_cls.SEDOL_PROPERTY_KEY, models.PropertyValue("B019KW7"))
+
+        instrument_definition4 = models.InstrumentDefinition(name="SAINSBURY (J) PLC",
+                                                             identifiers={test_cls.FIGI_SCHEME: "BBG000BF0KW3",
+                                                                          test_cls.CUSTOM_INTERNAL_SCHEME: "INTERNAL_ID_4"},
+                                                             properties=[isin_sains, sedol_sains])
+
+        isin_tayl = models.InstrumentProperty(test_cls.ISIN_PROPERTY_KEY, models.PropertyValue("GB0008782301"))
+        sedol_tayl = models.InstrumentProperty(test_cls.SEDOL_PROPERTY_KEY, models.PropertyValue("0878230"))
+
+        instrument_definition5 = models.InstrumentDefinition(name="TAYLOR WIMPEY PLC",
+                                                             identifiers={test_cls.FIGI_SCHEME: "BBG000BF4KL1",
+                                                                          test_cls.CUSTOM_INTERNAL_SCHEME: "INTERNAL_ID_5"},
+                                                             properties=[isin_tayl, sedol_tayl])
+
+        upsert_response = test_cls.client.upsert_instruments(
```

Comment:
You can assign self here using self.upsert_response = test_cls.... etc.

This allows you to remove self.instrument_response = upsert_response below and you can just use self.upsert_response in place of upsert_response and self.instrument_response

Reply:
done